src/client/visualizations/scatterplot/utils/get-plotting-data.mocha.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { expect } from "chai";
+import { Stage } from "../../../../common/models/stage/stage";
+import { extendExtentIfNeeded, getTicksForAvailableSpace, getXAxisLabelPosition } from "./get-plotting-data";
+
+describe("extendExtentIfNeeded", () => {
+  it("should extend when min and max are the same", () => {
+    const extent = [10, 10];
+
+    const result = extendExtentIfNeeded(extent);
+
+    expect(result).to.eql([9.5, 10.5]);
+  });
+
+  it("should not extend when min and max differ", () => {
+    const extent = [10, 100];
+
+    const result = extendExtentIfNeeded(extent);
+
+    expect(result).to.eql([10, 100]);
+  });
+});
+
+describe("getTicksForAvailableSpace", () => {
+  it("should return not changed ticks when screen size is larger than 768px", () => {
```

Comment:
Not "screen size" but general "available space". "when has more than 768px space available"?

Reply:
Yes, that's more accurate. Will fix!

Reply:
aefaeec ✅